packages/superset-ui-core/src/models/Registry.ts

```diff
@@ -108,26 +144,26 @@ export default class Registry {
     );
   }
 
-  keys() {
+  keys(): string[] {
     return Object.keys(this.items);
   }
 
-  values() {
+  values(): Array<V | Promise<V> | undefined> {
```

Comment:
looks like you can also do `(type | type | type)[]`, the `Array<..>` generic seems a little more readable for this case tho so 🤷‍♂️

Reply:
I will keep the `Array<>` as it is more readable. Will use `[]` for primitive and types.